src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        ColorSupplier supplier = new ColorSupplier();
+        Random random = new Random();
```

Comment:
they should be class-level
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level

Reply:
I already fixed it and asked for review, please check my new commit "add some fixes:..."

Reply:
should I create a new branch for review?